packages/flutter/test/cupertino/segmented_control_test.dart

```diff
@@ -893,7 +919,13 @@ void main() {
       ),
     );
 
-    await expectLater(find.byType(RepaintBoundary), matchesGoldenFile('segmented_control_test.0.0.png'));
+    // if golden files are regenerated on a non-Mac machine,
+    // change the 'skip' line to only run this test on that type of machine
+    await expectLater(
+      find.byType(RepaintBoundary),
+      matchesGoldenFile('segmented_control_test.0.0.png'),
+      skip: !Platform.isMacOS,
```

Comment:
Do we want to skip the entire test on non Mac machines, or just not make assertions?

Reply:
I think we should just not make the assertion. The 'skip' line should accomplish this.